src/ExcelProvider.Runtime/ExcelProvider.Runtime.fs

```diff
@@ -122,7 +122,7 @@ module internal ExcelAddressing =
 
                 topLeft,
                 { topLeft with
-                    Row = sheet.Rows.Count
+                    Row = sheet.Rows.Count - 1
                     Column = sheet.Columns.Count - 1 },
```

Comment:
In this case, a range from A1 to B3 would have rows 1,2,3 and columns A,B,C. Count for both would be 3. For columns the index of the bottomRight being defined here would correctly be 2, but the rows would be 3, grabbing an extra row.